lib/src/realtime_channel.dart

```diff
@@ -61,11 +61,9 @@ class ChannelFilter {
   }
 }
 
-enum ChannelResponse {
-  ok,
-  timedOut,
-  rateLimited;
+enum ChannelResponse { ok, timedOut, rateLimited }
 
+extension Name on ChannelResponse {
```

Comment:
I would rename the extension, because it's exported and may cause a conflict with another extension in the user's project.

Reply:
Come to think about it, I don't think we need to actually convert `ChannelResponse` to a string at all, so we can get rid of it all together.